### ISSUE TITLE
Use median (not mean) for Salary Breakdown

### ISSUE THREAD

**Issue by pwim** [SachaG/tokyodev#72]
Let's use median (50th) percentile instead of the mean (average), as it is better at minimizing the impact of outlier data, and is more appropriate given our small sample size and that we rely on user reported values.

This might have some duplication between the "Average Salary" chart and "Salary Percentiles", but perhaps having a simpler view than the box plot makes sense as a default.

**Comment by SachaG** [comment]
Yeah I was going to have the boxplot as the default once it's ready. But it's true we could also have the median instead of the average. Maybe we can decide once we have the boxplot to compare with? I agree the average is often misleading though.
